app/src/main/java/com/mapbox/dash/example/MainActivity.kt

```diff
@@ -289,6 +308,26 @@ class MainActivity : DrawerActivity() {
             val controller = Dash.controller
             controller.stopNavigation()
         }
+
+        menuBinding.cleanHistory.bindAction {
+            Dash.controller.cleanHistory()
+        }
+        menuBinding.addToHistory.bindAction {
+            Dash.controller.addHistoryItem(searchItem)
+        }
+        menuBinding.removeFromHistory.bindAction {
+            Dash.controller.removeHistoryItem(searchItem)
+        }
+        menuBinding.addToFavorites.bindAction {
+            Dash.controller.addFavoriteItem(favoriteItem)
+        }
+        menuBinding.removeFromFavorites.bindAction {
+            Dash.controller.removeFavoriteItem(searchItem)
```

Comment:
```suggestion
            Dash.controller.removeFavoriteItem(favoriteItem)
```
anyway it will work with both items, but let's use `favoriteItem`

Reply:
Fixed.